pkg/gofr/datasource/opentsdb/opentsdb.go

```diff
@@ -16,6 +16,13 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
+// Predefined static errors.
+var (
+	errInvalidResponseType = errors.New("invalid response type")
+	errInvalidQueryParam   = errors.New("invalid query parameters")
+	errInvalidParam        = errors.New("invalid parameter type")
+)
```

Comment:
These errors will be returned to clients

I would expect them to be exported

Reply:
these errors are wrapped and then returned to clients. So I feel they can stay unexported.

Reply:
It's open to debate, but OK